alembic/env.py

```diff
@@ -9,7 +9,7 @@
 
 from alembic import context
 from openadapt.config import DB_URL
-from openadapt.db import Base
+from openadapt.db.db import Base
```

Comment:
I'm not sure if I like `.db.db`. Maybe we can do `from openadapt.db import db`.

Reply:
oh true, vscode did these changes for me